Offseason2022/src/main/java/frc/robot/commands/TowerConveyorRun.java

```diff
@@ -29,6 +30,7 @@ public TowerConveyorRun(TowerConveyor towerConveyor, TCMode mode)
   public void initialize( )
   {
     m_towerConveyor.setTowerConveyorSpeed(m_mode);
+    DataLogManager.log("TowerCONVERYORSHOOTBALL");

```

Comment:
This is redundant with the ScoringActionUpperHub, and I don't think we want this message to go out every time the tower conveyor is turned on (cause it will in the initialize function of TowerConveyorRun). Let's not leave this checked in.

Also, when adding DataLog calls, let's make sure that we want them for long term debugging, and not just a short term peek into what's happening. We'll flood the log output if we aren't careful.